packages/SwingSet/package.json

```diff
@@ -34,6 +34,7 @@
     "@agoric/store": "^0.8.3",
     "@agoric/swing-store": "^0.8.1",
     "@agoric/swingset-liveslots": "^0.9.0",
```

Comment:
Does swingset still have a dependency on liveslots? I only see an import of `makeMarshaller` in tests, which qualifies it for a dev dependency.

Reply:
The dependency is on its way out, for sure, as the liveslots dep will be encapsulated by the worker-v1 dep. I think you're right that it can get downgraded immediately, I'll try it.

Reply:
oh, right, we still have local workers (`src/kernel/vat-loader/manager-local.js`), and they need swingset, so nope, it needs to stay in `dependencies`

(our chain only runs `vat-comms` in a local worker, and comms doesn't use liveslots, but the ability to run fully-fledged liveslots vats in a local worker is still a swingset feature)

Reply:
oh yeah I missed actual usages of `makeLiveslots`, my bad.

I suppose we should ultimately have a "local" worker too as its own package?